docs/versioning.rst

```diff
@@ -0,0 +1,118 @@
+**********
+Versioning
+**********
+
+
+Release support
+===============
+
+
+.. list-table::
+   :header-rows: 1
+
+   * - Release
+     - :ref:`Support level<versioning_support_levels>`
+     - Minimum Trace Agent
+   * - 0.x
+     - :ref:`Maintenance<versioning_support_maintenace>`
+     -
+   * - 1.x
+     - :ref:`General Availability<versioning_support_ga>`
+     - 7.28
+
+
+Versions
+========
+
+
+The non-negative integer components of the **version format** (``v<MAJOR>.<MINOR>.<PATCH>``) are incremented by the criteria:
+
+MAJOR
+    Incompatible changes to the public :ref:`interface<versioning_interfaces>`
+
+    Removing support for a :ref:`runtime<versioning_supported_runtimes>`.
+
+MINOR
+    Backwards compatible changes to the public :ref:`interface<versioning_interfaces>`
+
+    Any backwards compatible or incompatible changes to the internal :ref:`interface<versioning_interfaces>`
+
+    Adding a support for :ref:`runtime<versioning_supported_runtimes>`
+
+PATCH
+    Bug fixes that are backwards compatible
+
+    Security fixes that are backwards compatible
+
+.. _versioning_interfaces:
+
+Interfaces
+==========
+
+
+The definition of the **internal** interface:
+    The ``ddtrace.internal`` package is internal
```

Comment:
```suggestion
    The ``ddtrace.internal`` module is internal
```

Reply:
I think the `package` term is more apt here as it bundles modules together:

https://docs.python.org/3/tutorial/modules.html#packages